_tutorials/index.md

```diff
@@ -63,10 +63,11 @@ Simulation of external, viscous, incompressible flow around the NACA 0012 using
 Simulation of internal, laminar, incompressible flow over a backward-facing step with an inlet velocity profile input from file.
 * [Laminar Buoyancy-driven Cavity](/tutorials/Inc_Laminar_Cavity/)    
 Simulation of internal, laminar, incompressible flow in a differentially-heated cavity under the influence of gravity (classical natural convection case).
-* [Streamwise Periodicity](/tutorials/Inc_Streamwise_Periodic/)
+* [Streamwise Periodicity](/tutorials/Inc_Streamwise_Periodic/)    
```

Comment:
These added spaces trigger line break [here](https://su2code.github.io/tutorials/home/). Added for consistency